### ISSUE TITLE
Re-enable date taxonomy for custom post types

### ISSUE THREAD

**Issue by kevinross** [mtekk/Breadcrumb-NavXT#115]
In 104e680, support for date taxonomies was removed as wordpress didn't support them for CPTs at the time. As of at least WP3.9.3, they do. Can they be added back in?

**Comment by mtekk** [comment]
I'll look into this for 5.3.0.

**Comment by mtekk** [comment]
This one looks to be "easier said than done". While the option can be re-added, it appears there isn't a clean, reliable way to get the URL for each of the date archive. Tacking on ?post_type=posttypename could work, but that isn't necessarily clean.

**Comment by mtekk** [comment]
This is really a subset of #112

**Comment by kevinross** [comment]
Alright, I'll keep an eye on #112 then; I have a workaround good enough for my purposes for the moment but I try to avoid abusing filters this way :s

**Comment by mtekk** [comment]
The option was re-exposed in the admin interfaces (network and normal) in the gymnast branch. It won't do much in the backend yet, but the legs are there (depends on #112 for full support).

**Comment by mtekk** [comment]
This issues portion of #112 has been implemented
